### PR TITLE
refactor: create functionDetail.vue

### PR DESCRIPTION
**Pull request by aki-yuan**

### CODE REVIEW

**Comment by yezhidaizhu** [comment]
input 好像未处理哦，还有国际化

**Comment by yezhidaizhu** [comment]
mock数据似乎也没显示，及黄色警报过多，如果不能不处理，可以先处理国际化这部分（使用 vue- i18n中的t函数）及input部分

**Comment by aki-yuan** [comment]
> input 好像未处理哦，还有国际化

重构前的input就是无法改动的，未处理是指样式问题吗？

**Comment by yezhidaizhu** [comment]
好的，我可能是忘了

**Review by yezhidaizhu — APPROVED** [review]

File: front-end/src/views/function/components/FunctionDetail.vue [hunk 1]
```diff
@@ -0,0 +1,239 @@
+<template>
+  <el-drawer size="40%" @open="onOpen" @close="onClose">
+    <!-- function info -->
+    <el-form ref="info" v-loading="loadingDetail" :rules="rules" style="padding: 0 20px" :model="info"
+label-position="top">
```
Comment (RobertIndie) [inline front-end/src/views/function/components/FunctionDetail.vue:5]: Codacy has a fix for the issue: [Replace `label-position="top"` with `······label-position="top"⏎····`](https://app.codacy.com/gh/FunctionStream/function-stream/pullRequest?prid=7834760)
```suggestion
      label-position="top"
```

File: front-end/src/views/function/components/FunctionDetail.vue [hunk 2]
```diff
@@ -0,0 +1,239 @@
+<template>
+  <el-drawer size="40%" @open="onOpen" @close="onClose">
+    <!-- function info -->
+    <el-form ref="info" v-loading="loadingDetail" :rules="rules" style="padding: 0 20px" :model="info"
```
Comment (RobertIndie) [inline front-end/src/views/function/components/FunctionDetail.vue:4]: Codacy has a fix for the issue: [Replace `·ref="info"·v-loading="loadingDetail"·:rules="rules"·style="padding:·0·20px"` with `⏎······ref="info"⏎······v-loading="loadingDetail"⏎······:rules="rules"⏎······style="padding:·0·20px"⏎·····`](https://app.codacy.com/gh/FunctionStream/function-stream/pullRequest?prid=7834760)
```suggestion
    <el-form
```

**Pull request merged**